src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/CompilationTask.kt

```diff
@@ -472,7 +472,7 @@ private val Directories.incrementalData
  * Create a new [JvmCompilationTask] with sources found in the generatedSources directory. This should be run after
  * annotation processors have been run.
  */
-internal fun JvmCompilationTask.expandWithGeneratedSources(): JvmCompilationTask =
```

Comment:
Internal will be visible to tests if you pass this target to the test target as an associate dependency.

Reply:
going to make it public to avoid using 4f0b72806f3603b977138116549762c119ef632c

Reply:
Forgot about the `kt_bootstrap` layer that these sources are wrapped in.